nest/nest.py

```diff
@@ -1571,15 +1610,15 @@ def never_authorized(self):
     @property
     def invalid_access_token(self):
         try:
-            self._status
+            self._get("/")
             return False
         except AuthorizationError:
             return True
 
     @property
     def client_version_out_of_date(self):
         if self._product_version is not None:
-            self._bust_cache()
+            # self._bust_cache()
```

Comment:
I had to comment out line 203 as well: https://github.com/jkoelker/python-nest/pull/131/files#diff-370427bed7ee1992efb67f05d124ca6cR203

Reply:
Oh yeah, I didn't notice that